### PR TITLE
Maven

### PR DESCRIPTION
**Pull request by andytung**
*\* changed from maven to gradle. 

No need to really review.

All I did was was added a build.gradle and changed the package structure.

The package structure was the maven default.

Note that we do not own the domain mafia.mobcore.org

If you prefer maven, then below is the pom.xml

<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

```
<groupId>org.mobcore.mafia</groupId>
<artifactId>mafiagame</artifactId>
<version>1.0-SNAPSHOT</version>
<dependencies>
    <dependency>
        <groupId>junit</groupId>
        <artifactId>junit</artifactId>
        <version>4.11</version>
        <scope>test</scope>
    </dependency>
</dependencies>

<build>
    <plugins>
        <plugin>
            <groupId>org.apache.maven.plugins</groupId>
            <artifactId>maven-compiler-plugin</artifactId>
            <version>3.3</version>
            <configuration>
                <source>1.8</source>
                <target>1.8</target>
            </configuration>
        </plugin>
    </plugins>
</build>
```

</project>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,112 +0,0 @@
-**04.06.2015**
-
-_CHANGES:_
```
Comment (maknoon) [inline CHANGELOG.md:3]: the changelog is an important file! don't delete it

**Comment by maknoon** [comment]
nice work. just a couple of things:
- ~~you should be including the pom.xml in your commit so that other people can edit the build script, the only files that are supposed to be added to gitignore are the ones that get _generated_ in a build every time~~ wait nvm you want to use gradle build ok thats cool too
- i'm guessing `mvn package` is good but did you try configuring for the other lifecycle commands to see if it works? <- just apply what i said here to gradle
- can you revert the changelog, i'm not sure why it is being deleted?

other than that, looks good

**Comment by andytung** [comment]
Change log needs to be updated and maintained if you want to keep it.
'mvn package' works with the pom.xml and 'gradle build' works.

**Comment by maknoon** [comment]
ya updating the changelog is everybody's responsibility when something new happens. it's something that should be kept tho imo

**Pull request merged**